torch/distributed/tensor/parallel/api.py

```diff
@@ -94,6 +94,15 @@ def parallelize_module(  # type: ignore[return]
             return _parallelize_multihead_attn(module, device_mesh)
         elif _is_mlp_for_pairwise_parallel(module):
             return _parallelize_mlp(module, device_mesh, parallelize_plan)
+        elif isinstance(module, torch.nn.Dropout):
```

Comment:
I am wondering if directly inserting this logic in TP might be necessary or not.

My two cents are:
Let's make this optional for users or make this as an extension which user can pass in because what if today users all use pure DTensor ops or what if we want to have other special handling ops like this? (For example, softmax, layernorm, etc.)